main.go

```diff
@@ -1,7 +1,36 @@
 package main
 
-import "os"
+import (
+	"fmt"
+	"log"
+
+	"github.com/stellar/starbridge/integrations"
+	"github.com/stellar/starbridge/model"
+	"github.com/stellar/starbridge/transform"
+)
 
 func main() {
-	os.Exit(1)
+	txHash := "0x13070f64d40f22cd10c5bf9972767b67406ed3d818a50f82b1409289dcaa1aec"
+
+	modelTxEth, e := integrations.FetchEthTxByHash(txHash)
```

Comment:
top level sample program is very simple. This is a simple demo of the working components.

Program Output:
```
transaction fetched as modelTxEth:
Transaction[Chain=Ethereum, Hash=0x13070f64d40f22cd10c5bf9972767b67406ed3d818a50f82b1409289dcaa1aec, Block=11754308, SeqNum=39899447, IsPending=false, From=0x81b7E08F65Bdf5648606c89998A9CC8164397647, To=0x3A3fc7E8dA2e51015f562f5fe35cEEE08923Ed11, AssetInfo=AssetInfo[Code=ETH, ContractAddress=0x0000000000000000000000000000000000000000, Decimals=18], Amount=1000000000000000000, HasOriginalTx=true, HasAdditionalOriginalTx=true]


transaction converted to modelTxStellar:
Transaction[Chain=Stellar, Hash=, Block=0, SeqNum=0, IsPending=true, From=GB42JR56FDOVUR75LN2J2F6DARS7SDUYMYPETQ24TDGRBCCQCHS2M2Y7, To=GCBAA5476KARHPDSU6WFQTPXQOWX3QMXU4LF7JVZ2ZMWJ4OQEL7ZMV6G, AssetInfo=AssetInfo[Code=ETH, ContractAddress=GB42JR56FDOVUR75LN2J2F6DARS7SDUYMYPETQ24TDGRBCCQCHS2M2Y7, Decimals=7], Amount=10000000, HasOriginalTx=true, HasAdditionalOriginalTx=false]


StellarTx[SourceAccount=GDXLLPH23EHIFOQLO46X2WQQPBBRJ6YPV7JOXEWS7V3AU74Z4EY7PGCS, SeqNum=1, BaseFee=100, MaxFee=100, TimeBounds.MinTime=0, TimeBounds.MaxTime=0, Memo=, Operations=Array[Operation[SourceAccount=, Type=CreateClaimableBalance, Destinations=[Claimant[Desination=GCBAA5476KARHPDSU6WFQTPXQOWX3QMXU4LF7JVZ2ZMWJ4OQEL7ZMV6G, Predicate=ClaimPredicateTypeClaimPredicateUnconditional]], Amount=10000000, Asset=ETH:GB42JR56FDOVUR75LN2J2F6DARS7SDUYMYPETQ24TDGRBCCQCHS2M2Y7]]]
```